### PR TITLE
Improved installing-vpe.md

### PR DESCRIPTION
**Pull request by schuster-rainer**
Added more hints on installation and navigation

### CODE REVIEW

File: VisualPinball.Unity/Documentation~/creators-guide/setup/installing-vpe.md [hunk 1]
```diff
@@ -10,7 +10,7 @@ In order to start creating or modifying tables with VPE, the first thing you'll
 > [!NOTE]
 > As long as you don't use Unity for a game that makes $100K or more in revenue or funding a year, the free [Personal](https://store.unity.com/compare-plans) plan is sufficient for you.
 
-Unity uses an application called *Unity Hub* to update itself and provide quick access to your projects. The install process is straight-forward and documented [here](https://docs.unity3d.com/Manual/GettingStartedInstallingHub.html) if you run into troubles.
+Unity uses an application called *Unity Hub* to update itself, create new projects and provide quick access to them. Choose version 3 Beta or newer as we need it for later. The install process is straight-forward and documented [here](https://docs.unity3d.com/Manual/GettingStartedInstallingHub.html) if you run into troubles.
```
Comment (freezy) [inline VisualPinball.Unity/Documentation~/creators-guide/setup/installing-vpe.md:13]: You sure that Hub v3 is necessary?

File: VisualPinball.Unity/Documentation~/creators-guide/setup/installing-vpe.md [hunk 2]
```diff
@@ -38,7 +38,7 @@ Once the editor has opened you can click away the HDRP Wizard that opens. You sh
 
 Click on *File -> New Scene* and select the *Basic Indoors (HDRP)* template. Save it in your *Assets/Scenes* folder as `TestTable.unity`.
 
-In this base scene there's a plane that will [Z-fight](https://en.wikipedia.org/wiki/Z-fighting) with our imported table later. Select it in the *Hierarchy* and move it down a little by setting the *Y Position* under *Transform* in the *Inspector* to `-0.1`. Your scene should now look like this:
+In this base scene there's a *Plane* that will [Z-fight](https://en.wikipedia.org/wiki/Z-fighting) with our imported table later. Select it in the *Hierarchy* ( left by default) and move it down a little by setting the *Y Position* under *Transform* in the *Inspector* (right by default) to `-0.1`. Your scene should now look like this:
```
Comment (freezy) [inline VisualPinball.Unity/Documentation~/creators-guide/setup/installing-vpe.md:41]: If it's not clear enough, I'd prefer linking to Unity's doc ([hierarchy](https://docs.unity3d.com/Manual/Hierarchy.html) / [inspector](https://docs.unity3d.com/Manual/UsingTheInspector.html)).

**Review by freezy — COMMENTED** [review]
Hi, and thanks! :)

Comments above.

File: VisualPinball.Unity/Documentation~/creators-guide/setup/installing-vpe.md (hunk 1 above)
Comment (schuster-rainer) [reply VisualPinball.Unity/Documentation~/creators-guide/setup/installing-vpe.md:13]: No, not sure. I just assumed to use the latest as I saw it on one of the screenshots.

**Review by freezy — APPROVED** [review]
Thanks!

**Pull request merged**